src/main/java/athleticli/parser/Parser.java

```diff
@@ -185,6 +186,9 @@ public static Command parseCommand(String rawUserInput) throws AthletiException
      * @throws AthletiException If the input format is invalid.
      */
     public static LocalDateTime parseDateTime(String datetime) throws AthletiException {
+        if (datetime.startsWith(INVALID_YEAR)) {
+            throw new AthletiException(Message.MESSAGE_DATETIME_INVALID);
+        }
```

Comment:
Nice use of date time features! I suggest to create a more specific exception message for this case, as the input format is actually valid but the boundary is not.

Reply:
I feel this should be okay. The updated error message would read:

`The datetime must be valid and in the format yyyy-MM-dd HH:mm!`

Year 0000 is not valid datetime, which is conveyed by the new updated message.

Reply:
Yep that's good